tests/execution/cors.md

```diff
@@ -0,0 +1,45 @@
+# Cors
+
+#### server:
+
+```graphql
+schema
+  @server(
+    responseHeaders: [
+      {key: "Access-Control-Allow-Origin", value: "*"}
+      {key: "Access-Control-Allow-Headers", value: "*"}
+      {key: "Access-Control-Allow-Methods", value: "POST, GET, OPTIONS"}
+    ]
+  ) {
+  query: Query
+}
+
+type Query {
+  value: Int! @http(path: "/", baseURL: "http://api.com")
+}
+```
+
+#### mock:
+
+```yml
+- request:
+    method: POST
+    url: http://api.com
+    body: null
+  response:
+    status: 200
+    body:
+      value: 1
+```
```

Comment:
why is this mock required?

Reply:
removed the mock. using `@const` instead